style.css

```diff
@@ -2,7 +2,7 @@ html,
 body {
   min-width: 290px;
```

Comment:
- **[Optional]** In order for your comments to sound more human-friendly you can add some emojis to your comments 💪🏼🙋‍♂️